ibaqpy/ibaq/imputation_methods.py

```
@@ -0,0 +1,102 @@
+    if isinstance(data, pd.DataFrame):
+        # If it's a single DataFrame, transform it and return immediately
+        return pd.DataFrame(imputer.fit_transform(data), columns=data.columns, index=data.index)
+    else:
+        # Otherwise, use list comprehension to apply the imputer to each DataFrame
+        return [
+            pd.DataFrame(imputer.fit_transform(t), columns=t.columns, index=t.index) for t in data
+        ]
+
+
+
+def impute_missing_values(dataset_df, field, class_field):
```

Ruff (0.8.0)

Redefinition of unused `impute_missing_values` from line 9

(F811)